### PR TITLE
Add markdown and modify recruitment emails

### PR DESCRIPTION
**Pull request by celestemartinez**
Closes:
- #811 
- #810 

I changed recruit_reviewers email to be the same as the one used in the request form, which includes venue role in subject and message (previous message always said 'to serve as reviewer' even if ACs were being recruited which would cause confusion).

Added some formatting to deployment comment.

Added abstract to remove submission options.

### CODE REVIEW

File: openreview/conference/builder.py [hunk 1]
```diff
@@ -951,15 +951,15 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
 
         invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self, options)
         invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options())
-        recruit_message = '''Dear {name},
 
-        You have been nominated by the program chair committee of ''' + self.short_name + ''' to serve as a reviewer.  As a respected researcher in the area, we hope you will accept and help us make the conference a success.
+        role = 'reviewer' if reviewers_name == 'Reviewers' else 'area chair'
```
Comment (melisabok) [inline openreview/conference/builder.py:955]: How do we do this in the request form when the reviewers_name is 'Area_Chairs'? Do we need to replace underscores with spaces?

File: openreview/venue_request/venue_request.py [hunk 2]
```diff
@@ -24,8 +24,8 @@ def setup_venue_revision(self):
         }
         revision_content['remove_submission_options'] = {
             'order': 19,
-            'values-dropdown':  ['keywords', 'pdf', 'TL;DR'],
-            'description': 'Fields to remove from the default form: keywords, pdf, TL;DR'
+            'values-dropdown':  ['abstract','keywords', 'pdf', 'TL;DR'],
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:27]: Thanks for this!

File: openreview/conference/builder.py [hunk 3]
```diff
@@ -973,14 +973,15 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
...
+        If you have any questions, please contact info@openreview.net.
 
         Cheers!
 
         Program Chairs
 
         '''
-        recruit_message_subj = self.id + ': Invitation to Review'
+
+        recruit_message_subj = '[' + self.get_short_name() + ']: Invitation to serve as ' + role.capitalize()
```
Comment (melisabok) [inline openreview/conference/builder.py:984]: I like this new subject!

File: openreview/conference/builder.py (hunk 1 above)
Comment (celestemartinez) [reply openreview/conference/builder.py:955]: if reviewer_name == 'Area_Chairs', then role would be 'area chair', is this what you mean?

Comment (melisabok) [reply openreview/conference/builder.py:955]: I see, I think this is ok for this PR but we should translate the role from the group id so we can support other names than reviewers and area chairs.

**Review by melisabok — APPROVED** [review]

**Pull request merged**